test/e2e/adminapi_kubernetesobjects.go

```diff
@@ -0,0 +1,350 @@
+package e2e
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+)
+
+var _ = Describe("[Admin API] Kubernetes objects action", func() {
+	BeforeEach(skipIfNotInDevelopmentEnv)
+
+	var (
+		ctx        context.Context
+		resourceID string
+		namespace  string
+		objName    string
+	)
+
+	BeforeEach(func() {
+		// Reset all the values even if you have to set a zero value, because
+		// we want to avoid test pollution where we use values from previous runs
+		ctx = context.Background()
+		resourceID = resourceIDFromEnv()
+		namespace = "default"
+		objName = "e2e-test-object"
+	})
+
+	testSecretOperationsForbidden := func() {
+		It("should not be able to create a secret", func() {
+			By("creating a new secret via RP admin API")
+			body := fmt.Sprintf(
+				`{
+					"kind": "Secret",
+					"apiVersion": "v1",
+					"metadata": {
+						"name": "%s",
+						"namespace": "%s"
+					},
+					"type": "Opaque",
+					"data": {
+						"password": "MWYyZDFlMmU2N2Rm"
```

Comment:
don't know what this value is but it doesn't seem to be a necessary part of the test?